src/templates/mod.rs

```diff
@@ -0,0 +1,92 @@
+use askama::Template;
+use itertools::Itertools;
+
+use crate::search::SearchResult;
+use crate::{Message, ScrollDirection, Toc};
+
+#[derive(Template)]
+#[template(path = "toc.html")]
+pub struct TocTemplate<'a> {
+    toc: &'a Toc,
+}
+
+impl<'a> TocTemplate<'a> {
```

Comment:
Is there a better name we could assign to this.
Assuming Toc stands for "Table of Contents", but it is not super obvious, and I think it would be confusing to find the right HTML/Template processor for someone who does not have any experience with the code.
Maybe ChannelListTemplate or SidebarTemplate would be better (and renaming the html file as well of course)?

Reply:
Yeah, channel list sounds better tbh, since it's not a book.